app/src/main/AndroidManifest.xml

```diff
@@ -45,6 +45,10 @@
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme"
             android:windowSoftInputMode="adjustResize"/>
+        <activity
+            android:name=".screen.requestleave.ListRequestLeave.ListRequestLeaveActivity"
```

Comment:
WARNING: [InnerclassSeparator] Use '$' instead of '.' for inner classes (or use only lowercase letters in package names); replace ".screen.requestleave.ListRequestLeave.ListRequestLeaveActivity" with ".screen$requestleave$ListRequestLeave$ListRequestLeaveActivity"
 When you reference an inner class in a manifest file, you must use '$' instead of '.' as the separator character, i.e. Outer$Inner instead of Outer.Inner.

(If you get this warning for a class which is not actually an inner class, it's because you are using uppercase characters in your package name, which is not conventional.)